Makefile

```
@@ -0,0 +1,86 @@
+	@$(MAKE) worktree-bootstrap DIR="$(ORCA_WORKTREES)/fix-$(ISSUE)"
+	@echo ""
+	@echo "Next:"
+	@echo "  cd $(ORCA_WORKTREES)/fix-$(ISSUE) && pnpm install"
+
+# Wire harness into an existing worktree (or re-run after pull).
+worktree-bootstrap:
+	@test -n "$(DIR)" || (echo "error: DIR= required" >&2; exit 1)
+	@test -x "$(BOOTSTRAP)" || chmod +x "$(BOOTSTRAP)"
+	@ORCA_PRIMARY="$(ORCA_PRIMARY)" bash "$(BOOTSTRAP)" "$(abspath $(DIR))"
+
+worktree-rm:
```

checkmake (0.3.2)

[warning] Target body for "worktree-rm" exceeds allowed length of 5 lines (13).

(maxbodylength)